creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -126,7 +126,31 @@ const retrieveClockValueForUserFromReplica = async (replica, wallet) => {
   return clockValue
 }
 
+const processJob = async (
```

Comment:
siiiiick

realistically this is a generic bull util and likely should go in smth like `creator-node/src/utils/bull.js` but w/e

Reply:
I like that idea -- will look into as a followup so we can have improved logging for other queues in CN: https://linear.app/audius/issue/CON-192/use-processjob-to-pass-a-job-specific-logger-to-each-bull-processor

Reply:
love it